.travis.yml

```diff
@@ -4,6 +4,8 @@ matrix:
   include:
   - python: 2.7
     env: TOX_ENV=pep8
+  - python: 3.7
+    env: TOX_ENV=pep8
```

Comment:
This needs to be the tox env `py37` (don't use pep8 here)

Reply:
Resubmitted as 1 commit.